docs/src/configuration/services/mysql/_index.md

```diff
@@ -136,7 +136,6 @@ highlight=python
 ### Configure connections
 
 There may be cases where you want to configure a database connection manually.
-For example, when using a framework like Symfony Flex that expects the connection as an environment variable.
```

Comment:
As it's done automatically, the explanation below is not relevant for Symfony projects.